docs/modules/monitoring/con-prometheus-metrics-proxy.adoc

```diff
@@ -0,0 +1,101 @@
+// file included in the following:
+//
+// kroxylicious-operator/index.adoc
+
+[id='con-prometheus-metrics-proxy-{context}']
+= Overview of proxy metrics
+
+[role="_abstract"]
+
+The proxy provides *connection metrics* and *message metrics*.
+
+Connection metrics allow the connections made from the clients to the proxy (downstream)
+and the connection made by the proxy to the Kafka cluster (upstream) to be understood.
```

Comment:
```suggestion
The proxy provides metrics about *connections*, both incoming and out going, and *messages*.

The metrics are grouped into downstream, or client side, metrics and upstream, or broker side, metrics so that users can interpret the effect the proxy and its filters are having on their Kafka clusters.
```